Doc/library/socketserver.rst

```diff
@@ -93,9 +93,9 @@ synchronous servers of four types::
    +-----------+        +--------------------+
 
 Note that :class:`UnixDatagramServer` derives from :class:`UDPServer`, not from
-:class:`UnixStreamServer` --- the only difference between an IP and a Unix
-server is the address family, which is simply repeated in both Unix
-server classes.
+:class:`UnixStreamServer` --- the only difference between a TCP or UDP server
+and a Unix stream or datagram server is the address family, which is simply repeated in both Unix
+stream and datagram server classes.
```

Comment:
You don't need to say "both" and then repeat exactly what "both" means. Either go for _"in both Unix server classes"_ or _"in Unix stream and datagram server classes"_.

Reply:
Fixed.

Reply:
Now you introduced more repetition again. At this point I'd prefer to simply cut the sentence post the comma:

_"the only difference between a TCP or UDP server and a Unix stream or datagram server is the address family."_

I'm not sure spelling out _TCP_ and _UDP_ is very important; perhaps this is even better:

_"the only difference between an IP and a Unix server is the address family."_

Perhaps it is ok to leave it as it is, despite the small inconsistency; after all, it's been like that for a very long time.